travel-agency-service/src/main/java/cz/muni/fi/pa165/bluebat/service/TripServiceImpl.java

```diff
@@ -21,52 +17,69 @@
 
     private final TripDao tripDao;
 
-    private PriceService priceService;
-
-    private ExcursionService excursionService;
-
+    private final PriceService priceService;
 
     @Autowired
     public TripServiceImpl(TripDao tripDao,PriceService priceService) {
         this.tripDao = tripDao;
         this.priceService = priceService;
-
     }
 
     @Override
     public void create(Trip trip) {
         Validator.NotNull(trip,"Trip");
-        try{tripDao.create(trip);} catch (Exception ex){ throw new WrongDataAccessException("Wrong data access",ex);}
+
+        try {
+            tripDao.create(trip);
+        } catch (Exception ex) {
+            throw new WrongDataAccessException("Wrong data access", ex);
+        }
     }
 
     @Override
     public void update(Trip trip) {
         Validator.NotNull(trip,"Trip");
-        Trip previous = null;
-        try{previous = tripDao.findById(trip.getId());} catch (Exception ex){ throw new WrongDataAccessException("Wrong data access",ex); }
+        Trip previous;
 
-        Validator.Found(trip,"Trip");
-        priceService.updatePrices(previous.getPrices(), trip.getPrices());
-        try{tripDao.update(trip);} catch (Exception ex){ throw new WrongDataAccessException("Wrong data access",ex); }
+        try {
+            previous = tripDao.findById(trip.getId());
+            Validator.Found(trip,"Trip");
```

Comment:
Toto bych nekontroloval v try bloku. Pak by to vypadalo jako chyba DAO, ale chyba je v tom, že se předalo špatné id tripu

Reply:
pravda, fixed